src/pytest_postgresql/factories/process.py

```diff
@@ -151,118 +129,25 @@ def postgresql_proc_fixture(
         # start server
         with postgresql_executor:
             postgresql_executor.wait_for_postgres()
-
-            yield postgresql_executor
+            template_db_name = f"{postgresql_executor.db_name}_tmpl"
+            with DatabaseJanitor(
+                user=postgresql_executor.user,
+                host=postgresql_executor.host,
+                port=postgresql_executor.port,
+                db_name=template_db_name,
+                version=postgresql_executor.version,
+                password=postgresql_executor.password,
+            ):
+                if populate_template:
+                    for populate_function in populate_template:
```

Comment:
How many databases can be set up? Actually consider having a `Dict[str, Callable]` for `populate_template` and one function per dbname

Reply:
Not at the moment, that would require a bit more rethinking though